docs/model-dev-guide/model-management/checkpoints.rst

```diff
@@ -40,68 +40,68 @@ configuration file will result in checkpoints being sorted by the loss metric in
+descending order with respect to the specified metric.
 
 .. code:: python
 
-   from determined.experimental import client
+   from determined.experimental import checkpoint, client
 
-   checkpoint = (
-       client.get_experiment(id).top_checkpoint(sort_by="accuracy", smaller_is_better=False)
+   checkpoints = (
+       client.get_experiment(id).list_checkpoints(
+           sort_by="accuracy",
+           order_by=checkpoint.CheckpointOrderBy.DESC
+       )
    )
 
-You may also query multiple checkpoints at the same time using the
-:meth:`~determined.experimental.client.ExperimentReference.top_n_checkpoints` method. Only the
-single best checkpoint from each trial is considered; out of those, the checkpoints with the best
-validation metric values are returned in sorted order, with the best one first. For example, the
-following snippet returns the top five checkpoints from distinct trials of a specified experiment.
+You may also query checkpoints by preset checkpoint parameters, as defined by
+:class:`~determined.experimental.checkpoint.CheckpointSortBy`. The following code snippet fetches
+all checkpoints for the experiment sorted by the checkpoint's trial ID in descending order.
 
 .. code:: python
 
-   from determined.experimental import client
-
-   checkpoints = client.get_experiment(id).top_n_checkpoints(5)
-
-This method also accepts ``sort_by`` and ``smaller_is_better`` arguments.
+   from determined.experimental import checkpoint, client
 
-:class:`~determined.experimental.client.TrialReference` is used for fine-grained control over
-checkpoint selection within a trial. It contains a
-:meth:`~determined.experimental.client.TrialReference.top_checkpoint` method, which mirrors
-:meth:`~determined.experimental.client.ExperimentReference.top_checkpoint` for an experiment. It
-also contains :meth:`~determined.experimental.client.TrialReference.select_checkpoint`, which offers
-three ways to query checkpoints:
+   checkpoints = client.get_experiment(id).list_checkpoints(
+       sort_by=checkpoint.CheckpointSortBy.TRIAL_ID,
+       order_by=checkpoint.CheckpointOrderBy.DESC
+   )
 
-#. ``best``: Returns the best checkpoint based on validation metrics as discussed above. When using
-   ``best``, ``smaller_is_better`` and ``sort_by`` are also accepted.
-#. ``latest``: Returns the most recent checkpoint for the trial.
-#. ``uuid``: Returns the checkpoint with the specified UUID.
+:class:`~determined.experimental.client.Trial` is used for fine-grained control over checkpoint
+selection within a trial. It contains a
+:meth:`~determined.experimental.client.Trial.list_checkpoints` method, which mirrors
+:meth:`~determined.experimental.client.Experiment.list_checkpoints` for an experiment.
 
-The following snippet showcases how to use the different modes for selecting checkpoints.
+The snippet below demonstrates various ways of selecting a checkpoint from a trial.
```

Comment:
The following code illustrates methods to select specific checkpoints from a trial: